pkg/skaffold/runner/v1/deploy.go

```diff
@@ -102,19 +102,14 @@ See https://skaffold.dev/docs/pipeline-stages/taggers/#how-tagging-works`)
 	ctx, endTrace := instrumentation.StartTrace(ctx, "Deploy_Deploying")
 	defer endTrace()
 
-	// we only want to register images that are local AND were built by this runner
+	// we only want to register images that are local AND were built by this runner OR forced to load via flag
 	var localAndBuiltImages []graph.Artifact
 	for _, image := range localImages {
-		if r.wasBuilt(image.Tag) {
+		if r.runCtx.ForceLoadImages() || r.wasBuilt(image.Tag) {
```

Comment:
nit: might make sense to add a test case to pkg/skaffold/runner/v1/deploy_test.go for ForceLoadImages to verify it works as intended

Reply:
@aaron-prindle I added an integration test in the previous PR: https://github.com/GoogleContainerTools/skaffold/pull/6428/files#diff-47686e4ae74b63544c96b8b7517bd27995b338c0ab343cad99f9bd3bad06d4f9R90

I wasnt sure how to update the unit tests to check the data I need, any hints or example code I can reference?